website/docs/reference/opspec/op-directory/op/parameter/number.md

```diff
@@ -18,3 +18,20 @@ A number to use as the value of the parameter when no argument is provided.
 
 ### isSecret
 A boolean indicating if the value of the parameter is secret. This will cause it to be hidden in UI's for example. 
+
+## Example
+
+This op echos a number parameter, defaulting to 9001, when run.
+
+```yaml
+name: example
+description: an example op
```

Comment:
Just a passing thought; you could consider moving the description to the op description; I don't know if this would actually work well or not though.